clang/include/clang/Parse/Parser.h

```diff
@@ -3101,6 +3101,11 @@ class Parser : public CodeCompletionHandler {
   std::optional<AvailabilitySpec> ParseAvailabilitySpec();
   ExprResult ParseAvailabilityCheckExpr(SourceLocation StartLoc);
 
+  void ParseAtomicAttribute(IdentifierInfo &AttrName,
```

Comment:
I'm shocked this is necessary, and would assume that most of this happens for us with normal statement attributes?  ALso, if we use `VariadicEnumArgument` most of the parsing stuff here becomes unnecessary?

Reply:
will use VariadicEnumArgument